LocationServices/LocationServices/Scenes/About/Sub-Scenes/Attribution/Controller/AttributionVC.swift

```diff
@@ -106,11 +106,22 @@ final class AttributionVC: UIViewController {
         }
     }
     
+    private let navBarNeeded: Bool
+    
+    init(navBarNeeded: Bool) {
+        self.navBarNeeded = navBarNeeded
+        super.init(nibName: nil,
+                   bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
There should be a constant string already.

Reply:
fixed